tools/test_live_repair.sh

```diff
@@ -26,11 +26,22 @@ mkdir -p "$REGION_ROOT"
 
 # Location of logs and working files
 WORK_ROOT=${WORK_ROOT:-/tmp}
-mkdir -p "$WORK_ROOT"
+TEST_ROOT="$WORK_ROOT/test_live_repair"
+if [[ ! -d "$TEST_ROOT" ]]; then
+    mkdir -p "$TEST_ROOT"
+    if [[ $? -ne 0 ]]; then
+        echo "Failed to make test root $TEST_ROOT"
+        exit 1
+    fi  
+else
+    # Delete previous test data
+    rm -r "$TEST_ROOT"
```

Comment:
Same concern as above (presumably this will also repeat below, so I'm not going to comment on subsequent files)